core/src/avm1/globals/system.rs

```diff
@@ -348,7 +339,22 @@ impl SystemProperties {
         percent_encoding::utf8_percent_encode(s, percent_encoding::NON_ALPHANUMERIC).to_string()
     }
 
-    pub fn get_server_string(&self, avm: &mut Avm1) -> String {
+    pub fn screen_resolution(&self, context: &UpdateContext) -> (u32, u32, f64) {
```

Comment:
(Nit: Would a documentation comment indicating which unit it is measured in be helpful?

Like, in case this is "pixels" (a guess on my part, I do not know what the unit is, and "pixels" is not necessarily all that accurate):

```rust
    /// Screen resolution, measured in pixels.
    pub fn screen_resolution(&self, context: &UpdateContext) -> (u32, u32, f64) {
```

?)

Reply:
(Nit: I imagine that, in a future PR (not in this PR), it might be nice to define some basic type aliases ( https://doc.rust-lang.org/reference/items/type-aliases.html ) for different types like this, to decrease need for documentation and for making the meaning of the types clearer. But this would again be in a future PR from someone).